Podfile

```diff
@@ -4,7 +4,7 @@ target 'HeroExamples' do
   platform :ios, '9.0'
   use_frameworks!
   pod "CollectionKit"
-  pod 'ChameleonFramework/Swift', :git => 'https://github.com/ViccAlexander/Chameleon.git'
+  pod 'ChameleonFramework/Swift', :git => 'https://github.com/wowansm/Chameleon.git', :branch => 'swift5'
```

Comment:
Should wait until mainline is swift5 compatible if not already.

Reply:
Project is no longer maintened. We should remove it from the examples.

Reply:
Makes sense to me, wanna just remove it within this PR then?